homeassistant/components/generic/camera.py

```diff
@@ -66,10 +68,15 @@ async def async_setup_platform(hass, config, async_add_entities, discovery_info=
     """Set up a generic IP Camera."""
 
     await async_setup_reload_service(hass, DOMAIN, PLATFORMS)
```

Comment:
We can remove the reload service since they would now reload the config entries

We need to import their existing cameras here so they don't have to set everything back up again in the UI


Looks for SOURCE_IMPORT in the code for examples

Reply:
Ok.  This will take me a bit longer to figure out.  Conflicts should be sorted now, but still working on your other requested changes.

Reply:
Part way through this.  I agree that this should be done, but it creates quite a lot of new issues.  Most of the tests in test_camera.py now fail because they use http://example.com, which fails the still_image/stream validation within the new config_flow.

I started adding patches to all of those tests.  I think it will work eventually.

But is this the correct approach?  Maybe it is better to just ignore the validation when importing configuration.yaml.  I am not sure what the recommended behaviour is in this scenario.

If no other recommendations, I will continue patching.

Reply:
All patches are done now and the tests pass.  I have one more error that occurs on my setup so setting to draft until I fix that.

Reply:
Usually we do the validation on import.  Patching is usually fine in this case.

Reply:
All errors fixed now.